web/package.json

```diff
@@ -20,57 +21,54 @@
     "ci": "yarn install --frozen-lockfile"
   },
   "private": true,
-  "resolutions": {
-    "webpack": "5.21.2"
-  },
   "dependencies": {
-    "@angular/animations": "11.1.0",
-    "@angular/cdk": "11.1.0",
-    "@angular/common": "11.1.0",
-    "@angular/core": "11.1.0",
-    "@angular/forms": "11.1.0",
-    "@angular/material": "11.1.0",
-    "@angular/platform-browser": "11.1.0",
-    "@angular/platform-browser-dynamic": "11.1.0",
-    "@angular/router": "11.1.0",
-    "@circlon/angular-tree-component": "10.0.2",
-    "@ngxs/store": "3.7.1",
-    "angular-svg-icon": "11.0.0",
+    "@angular/animations": "12.0.5",
+    "@angular/cdk": "12.0.5",
+    "@angular/common": "12.0.5",
+    "@angular/core": "12.0.5",
+    "@angular/forms": "12.0.5",
+    "@angular/material": "12.0.5",
+    "@angular/platform-browser": "12.0.5",
+    "@angular/platform-browser-dynamic": "12.0.5",
+    "@angular/router": "12.0.5",
+    "@circlon/angular-tree-component": "11.0.4",
+    "@ngxs/store": "3.7.2",
+    "angular-svg-icon": "12.0.0",
     "chart.js": "2.9.4",
```

Comment:
version 3.3.2 exists: https://www.npmjs.com/package/chart.js

Reply:
I think chart.js isn't installed directly but as dependency of ng2-charts which has not as up-to-date version

Reply:
Then it is a transitive dependency? -> remove?

Reply:
@cnguyen-de you are right: https://github.com/valor-software/ng2-charts/blob/2.4.2/package.json

Reply:
The install warnings indicate, that chart.js is a required peer dependency, so it needs to stay

Reply:
should we comment that? :)